conf/application.conf

```diff
@@ -147,6 +147,7 @@ urls {
   declareTransitMovementUnloadingFrontend = "http://localhost:9488/manage-transit-movements-unloading-remarks"
   declareTransitMovementArrivalFrontend = "http://localhost:9483/manage-transit-movements-arrivals"
   declareTransitMovementDepartureFrontend = "http://localhost:9489/common-transit-convention-departure"
+  declareTransitMovementCancellationFrontend = "http://localhost:9495/common-transit-convention-cancel-departure"
```

Comment:
Reminder: to add to app-config-base also

Reply:
:+1: